Sources/MapboxMaps/Annotations/ViewAnnotation.swift

```diff
@@ -66,6 +67,9 @@ public final class ViewAnnotationManager {
     ///   -  `ViewAnnotationManagerError.geometryFieldMissing` if options did not include geometry
     ///   - `MapError`: errors during insertion
     public func add(_ view: UIView, options: ViewAnnotationOptions) throws {
+        guard idsByView[view] == nil else {
+            return
```

Comment:
Should this be an error? What if the options passed this time are different from before?

Reply:
I was thinking about that too, but thought it might want it to behave similarly to `UIView.addSubview()` which also doesn't fail if we do it twice. But then again if we signal an error it might be more clear for the user in case they try different options.

Reply:
I'll add an error, since it's already a throwing function.